packages/node/src/lib/events/events.interface.ts

```diff
@@ -15,7 +15,7 @@ import {
 } from '@novu/shared';
 
 export interface IBroadcastPayloadOptions {
-  payload: ITriggerPayload;
+  payload?: ITriggerPayload;
```

Comment:
In `@novu/node`, the `payload` is marked as optional in all cases, because the payload schema is not known.

The API already accepts `payload: undefined`, so there's no need for any further changes in `@novu/node`.